tests/integration/test_schema_cache.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from app import cache, settings
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestApplicationVariables(IntegrationTestCase):
+
+    def setUp(self):
+        settings.EQ_DEV_MODE = False
+
+        for key_name, dev_location in settings._KEYS.items():
```

Comment:
Would we be able to apply this to all of the integration tests to run closer to production?